src/mongo/commands.ts

```diff
@@ -74,6 +75,13 @@ export class MongoCommands {
 		}
 	}
 
+	private static validateCollectionName(key: string): string | undefined | null {
+		if (/[#?\\\/\$]+/.test(key)) {
+			return "Cannot contain these characters - ?,#,\\,/, $ etc."
```

Comment:
'$' is a valid character. These are the only disallowed characters I get in the portal:
![screen shot 2017-11-06 at 9 44 52 am 2](https://user-images.githubusercontent.com/11282622/32455635-ae92eff0-c2d7-11e7-9251-342fa39347c4.png)

Reply:
[The mongo website](https://docs.mongodb.com/manual/reference/limits/#naming-restrictions) suggests otherwise